batch_job/src/#batch_job_hadoop.c#

```diff
@@ -1,3 +1,8 @@
+/*
```

Comment:
Is the name between #...# on purpose?
If it is not used, why are we keeping the file?